docs/pages/foundations/messaging/a_messaging_story.js

```diff
@@ -0,0 +1,202 @@
+      flex="grow"
+      justifyContent="center"
+      alignItems="center"
+    >
+      <Box maxWidth={format === 'oneCol' ? '100%' : 320} width="100%">
+        <Image alt="collage" src={imageUrl} naturalWidth={imageWidth} naturalHeight={imageHeight} />
+      </Box>
+    </Box>
+  );
+  const contentMobile = (
+    <Box width="100%" padding={4} lgDisplay="none" display="block">
+      <Flex direction="column" gap={2}>
+        <Heading size="400">{heading}</Heading>
+        <Text>{text}</Text>
+      </Flex>
+    </Box>
+  );
+  const contentDesktop = (
+    <Box width="100%" padding={4} lgDisplay="block" display="none">
+      <Flex direction="column" gap={2}>
+        <Heading size="400">{heading}</Heading>
+        <Text>{text}</Text>
+      </Flex>
+    </Box>
+  );
+  const lockup =
+    format === 'twoColImgRight' ? (
+      <React.Fragment>
+        {contentDesktop}
+        {img}
+        {contentMobile}
+      </React.Fragment>
+    ) : (
+      <React.Fragment>
+        {img}
+        {contentDesktop}
+        {contentMobile}
+      </React.Fragment>
+    );
+  return (
+    <Box
+      display="flex"
+      direction="column"
+      lgDirection={format === 'oneCol' ? 'column' : 'row'}
+      alignItems="center"
+    >
+      {lockup}
+    </Box>
+  );
+}
+
+export default function MessagingStoryPage(): Node {
+  return (
+    <Page title="A messaging story">
+      <PageHeader
+        name="A messaging story"
+        type="guidelines"
+        description="A brief story of how messages work together along with other components and patterns based on practical use cases from Pinterest products."
+      />
+
+      <Flex direction="column" gap={8}>
+        <Box color="infoBase">
+          <Flex alignItems="center">
+            <Box width="100%" padding={8}>
+              <Flex direction="column" gap={2}>
+                <Heading color="light" size="500">
+                  Meet Claire
+                </Heading>
+                <Text color="light">
+                  Claire Ọyáwálé is a Pinner who loves architecture and fine art. She also has a
+                  Pinterest Business account through her employer—a high-end shoe brand.
+                </Text>
+              </Flex>
+            </Box>
+            <Box width="40%" marginEnd={-1} marginBottom={-1}>
+              <Image
+                alt="collage"
+                src="https://i.pinimg.com/originals/fb/94/a7/fb94a7630c03b9542a3ce732940f4a6f.png"
+                naturalWidth={1388}
+                naturalHeight={1687}
+              />
+            </Box>
+          </Flex>
+        </Box>
+        <StoryItem
+          format="twoColImgLeft"
+          heading="Alert indicator"
+          text={`
+Claire opens Pinterest on her phone to see a red dot on her Avatar.
+
+This isn’t a highly urgent issue, so we don’t throw a modal, banner or other message component in her face just yet; just a little nudge that tells her she needs to look at her profile.
```

Comment:
Major nit (and I'm not a writer), but "throwing in her face" seems a bit harsh wording? Maybe:

"This isn’t a highly urgent issue, so we don’t display a prominent modal, banner or other message component just yet; just a little nudge that tells her she needs to look at her profile."

Reply:
Done!